app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
```

Comment:
- [ ] Kindly make your sign-up page look like that of the wireframe provided for this project. 

### Your Sign Up Page

![Sign-in](https://user-images.githubusercontent.com/77106962/162333754-5bb2a974-78d5-4de3-91da-4f972337772e.png)

### Wireframe Sign Up/ Log In Page

![image](https://user-images.githubusercontent.com/77106962/162333548-028aafe3-bcff-436e-bd9f-e094673db53a.png)

#### Note: You can keep the links under the _**signup**_ button but style them accordingly. 👍